build.ps1

```diff
@@ -185,5 +233,5 @@ if ($bypassPackaging){
     BuildPackages "" $false
 
     #build win-x86 extensions
-    BuildPackages "win-x86" $false
+    BuildPackages "win-x86" $true
```

Comment:
So this will effectively change the bits we're running in prod to no longer use the shared runtime, correct?

Reply:
yes hence my comment about win-x64 or else 64bit V2 functions will be broken

Reply:
As now we are refer to 2.1 official release we can use "shared" model. Self-contained param is removed.